### ISSUE TITLE
Unphysical Temperature Output in Partially-Adiabatic CMD Simulation

### ISSUE THREAD

**Issue by jorgecastro316** [i-pi/i-pi#336]
I am encountering an issue with the output temperature of the centroid normal mode in my partially-adiabatic CMD. The simulation setup utilizes a two-temperature PILE-G thermostat. While the overall simulation runs as expected and the results seem physically reasonable, when the **fixcom** flag is set to **True**, the temperature of the centroid normal mode, as obtained by the **get_temp** function in **properties.py**, is unphysically high, which seems inconsistent with the rest of the simulation results.

**Comment by venkatkapil24** [comment]
is this an issue only with partially adiabatic simulations? Can you check it for standard PIMD? What system are you running?

**Comment by jorgecastro316** [comment]
No. It also happens for the centroid mode temperature  whenever I use PILE-G thermostat, in single or double temperature mode. I am running MAPI and Carboxilic Acid Fluoride

**Comment by venkatkapil24** [comment]
it would be useful t have more info e.g. the centroid and higher normal mode temperatures (expected compared to observed).

**Comment by mahrossi** [comment]
Hi @jorgecastro316 , could you check whether the problem persists after the latest cleanup? (latest version of i-PI). If so, can you share inputs/outputs from completeness?

**Comment by mahrossi** [comment]
Related to #414 and to be addressed hopefully soon!

**Comment by jorgecastro316** [comment]
Sadly the problem persist. I have been circumventing this issue and getting the correct temperature for the nm=0 if and only if i set the fixcom=False. Otherwise I get a value one order of magnitude larger than expected. This was done with the latest version of i-pi :/. Here are the inputs for a water monomer
[for_ipi.zip](https://github.com/user-attachments/files/18503754/for_ipi.zip)

**Comment by mahrossi** [comment]
I think I will close this issue as it has been addressed with the latest devs by @venkatkapil24 and @weifang2321 .

@jorgecastro316 : can you please check that with the latest i-PI release you have none of these problems anymore? If any still remain, open a new issue with a full description, inputs and outputs. thanks!
